canopy/app/src/App.js

```diff
@@ -48,6 +52,18 @@ function App() {
   useEffect(() => {
     (async () => {
       const { userSaplingsResponse } = await loadAllSaplings();
+
+      const requiredUserSaplings = ['profile'];
```

Comment:
It may make more sense to have this a config sapling than a user sapling.

I can see the logic in putting in user sapling since it registers for a namespace, but perhaps we need to just add an optional namespace to the configSapling to handle this.

User saplings are meant to represent saplings that are installed per user and profile seems like something that would be configured at a global scope.

Reply:
Hmm, I think the name "config saplings" may not be descriptive of their purpose. Maybe we should call them "global saplings"? Maybe there should be a distinct name for sapling that do not register for a namespace, and run in the background?